.claude/CLAUDE.md

```diff
@@ -0,0 +1,256 @@
+# ZenML Claude Code Guidelines
+
+This document provides guidance for Claude Code when working with the ZenML codebase. ZenML is an extensible, open-source MLOps framework for creating production-ready ML pipelines.
+
+## Project Structure
+
+- `/src/zenml/` - Core source code
+  - `/artifact_stores/` - Artifact storage implementations
+  - `/cli/` - Command-line interface
+  - `/config/` - Configuration system
+  - `/container_registries/` - Container registry implementations 
+  - `/image_builders/` - Image building functionality
+  - `/integrations/` - External tool integrations
+  - `/materializers/` - Artifact serialization/deserialization
+  - `/model/` - Model-related functionality
+  - `/model_deployers/` - Model deployment implementations
+  - `/model_registries/` - Model registry implementations
+  - `/orchestrators/` - Pipeline execution orchestrators
+  - `/pipelines/` - Core pipeline functionality
+  - `/services/` - Services implementation
+  - `/stack/` - Stack components management
+  - `/steps/` - Pipeline step definitions
+  - `/utils/` - Utility functions
+  - `/zen_stores/` - Storage implementations
```

Comment:
I do not know how to put it myself but I wouldn't call this `Storage implementations`.